components/workspace/Panel.tsx

```diff
@@ -50,13 +51,16 @@ export function WorkspacePanel(props: PropsOf<typeof ActionPanel>) {
   const actionSubjects = useActionSubjects()
   const dropzoneRef = createRef<DropzoneRef>()
 
-  const handleRemoveSelected = () => {
-    const selectedItems = Array.from(
+  const getSelectedItems = () => {
+    return Array.from(
       formRef.current?.querySelectorAll<HTMLInputElement>(
         'input[type="checkbox"][name="workspaceItem"]:checked',
       ) || [],
     ).map((checkbox) => checkbox.value)
```

Comment:
Did you know: the second argument of `Array.from` is a map function ;-)

```suggestion
    return Array.from(
      formRef.current?.querySelectorAll<HTMLInputElement>(
        'input[type="checkbox"][name="workspaceItem"]:checked',
      ) || [],
      (checkbox) => checkbox.value
    )
```

Reply:
nope. TIL!